src/staging/src/templates/park.js

```diff
@@ -519,6 +519,7 @@ export const query = graphql`
           icon
           iconNA
           rank
+          defaultDescription
```

Comment:
Does this need to be on the site template too?

Reply:
I believe so. For example, this site [Wells Gray Park: Murtle Lake](https://bcparks.ca/wells-gray-park/murtle-lake/) has an empty kayak accordion.

Reply:
I believed I DID but I DIDN'T 😅 Updated the site too.